stacks/prod/service-accounts/service-account.tm.hcl

```diff
@@ -10,7 +10,7 @@
 globals {
 
   # The default acount ID to use: the stacks directory basename
-  sa_account_id = global.stack_basename
+  sa_account_id = "${global.stack_basename}-${global.environment}"
```

Comment:
as we use different projects there is no need to suffix environment.

Reply:
are we going to force people who want to run the examples to always create 2 projects and each having the staging/prod sufffix ? it seemed like a good idea to allow people to use a single project too to experiment.

WDYT @soerenmartius @i4ki

Reply:
yeah I kind of see that - should we really enforce using two projects?

Reply:
no strong objection tho